jwst/jump/tests/test_detect_jumps.py

```diff
@@ -447,8 +445,158 @@ def test_onecr_50_groups(setup_inputs):
     out_model = detect_jumps(model1, gain, rnModel, 4.0,  1, 200, 10, False)
     assert (out_model.groupdq[0, 5, 5, 5] == 4) # CR in group 5
     assert (out_model.groupdq[0, 30, 5, 5] == 4) # CR in group 30
-    assert (np.all(out_model.groupdq[0, 6:30, 5, 5] == 0)) # groups inbetween are not flagged
+    assert (np.all(out_model.groupdq[0, 6:30, 5, 5] == 0)) # groups in between are not flagged
+
+
+def test_single_CR_neighbor_flag( setup_inputs):
+    """"
+    A single CR in a 10 group exposure. Tests that:
+    - if neighbor-flagging is set, the 4 neighboring pixels *ARE* flagged, and
+    - if neighbor-flagging is *NOT* set, the 4 neighboring pixels are *NOT* flagged
+    """
+    grouptime = 3.0
+    ingain = 5
+    inreadnoise = np.float64(7)
+    ngroups = 10
+
+    model1, gdq, rnModel, pixdq, err, gain = \
+        setup_inputs( ngroups=ngroups, nrows=5, ncols=6, gain=ingain, readnoise=inreadnoise,
+                      deltatime=grouptime )
+
+    # two segments perfect fit, second segment has twice the slope
```

Comment:
What is the reason for having twice the slope after the jump?  I realize that's done in the other tests in test_detect_jumps.py, but I don't see the point.

Reply:
This test is Mike Regan's; I also don't understand the point and will look into this.